Bottom: 6f801eb7eafa041946dd433917a5aff6825ca50a
Top:    29adbe5f1a66820f89a7c0b003cad74f397b612f
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-05 13:48:16 -0500

Update docker-entrypoint to call parent container's entrypoint

After updating the parent image, our entrypoint wasn't setting some environment
variables correctly. Using the parent's entrypoint script after our setup
fixes this problem.


---
diff --git a/docker/docker-mediawiki/docker-entrypoint.sh b/docker/docker-mediawiki/docker-entrypoint.sh
index 0e8fc7e2..ff93cb5c 100755
--- a/docker/docker-mediawiki/docker-entrypoint.sh
+++ b/docker/docker-mediawiki/docker-entrypoint.sh
@@ -108,10 +108,8 @@ case ${1} in
     wiki)
         # verify permissions
         chown -R www-data:www-data ${WIKI_BASE_DIR}
-        # Apache gets grumpy about PID files pre-existing
-        rm -f /var/run/apache2/apache2.pid
         # Start apache
-        exec apache2 -D FOREGROUND
+        exec apache2-foreground
         ;;
 
     LocalSettings)
